JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -65,7 +66,8 @@ module Microsoft.ApplicationInsights {
                 emitLineDelimitedJson: () => this.config.emitLineDelimitedJson,
                 maxBatchSizeInBytes: () => this.config.maxBatchSizeInBytes,
                 maxBatchInterval: () => this.config.maxBatchInterval,
-                disableTelemetry: () => this.config.disableTelemetry
+                disableTelemetry: () => this.config.disableTelemetry,
+                sampleRate: () => this.config.samplingPercentage
```

Comment:
in schema - sampleRate which doesn't make sense, so it's samplingPercentage in config. I can't change schema. :)